src/config.ts

```diff
@@ -12,6 +13,28 @@ const AWSX_PROFILE_PATH = path.join(AWSX_HOME, 'profiles');
 const AWS_CREDENTIALS_PATH = path.join(AWS_HOME, 'credentials');
 const AWS_CONFIG_PATH = path.join(AWS_HOME, 'config');
 
+const fileCheck = (): void => {
+  try {
+    const missingFiles: string[] = [];
+
+    !fs.existsSync(AWSX_PROFILE_PATH) && missingFiles.push(AWSX_PROFILE_PATH);
+    !fs.existsSync(AWS_CREDENTIALS_PATH) && missingFiles.push(AWS_CREDENTIALS_PATH);
+    !fs.existsSync(AWS_CONFIG_PATH) && missingFiles.push(AWS_CONFIG_PATH);
+
+    if (missingFiles.length > 0) {
+      throw missingFiles;
```

Comment:
We should throw an error instead of an array of strings. You can just print the error message for the missing config files in this function and then throw an error.